tests/suit/framework/ova.go

```diff
@@ -17,7 +17,7 @@ func (r *OvaClient) LoadSourceDetails() (vm *OvaVM, err error) {
 		r.storageClass = "nfs-csi"
 	}
 
-	r.vmData.testVMId = "c5686650854d1e69b4123f4bf2e70fe1ed2a"
+	r.vmData.testVMId = "fdb4680df9160052f35e66c932e3fe5b62f5"
```

Comment:
need to look at our options and see if we cannot preserve those IDs

Reply:
that might be done if we are going to make them in another way. which means, it might change them anyway

Reply:
handled in #884